### PR TITLE
Add php 7.1 to travis

### PR DESCRIPTION
**Pull request by chalasr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,9 +8,10 @@ env:
 matrix:
     fast_finish: true
     include:
+        - php: 5.6
```
Comment (ogizanagi) [inline .travis.yml:11]: For the record: I moved this one below in order for Travis to execute the fastest and most complete build fist (7.0). But actually, it doesn't seem the order has any effect :/

File: .travis.yml [hunk 2]
```diff
@@ -8,9 +8,10 @@ env:
...
         - php: 7.0
           env: CHECK_PHP_SYNTAX="yes"
```
Comment (ogizanagi) [inline .travis.yml:13]: There are chances the php 7.1 build is slightly faster than 7.0. I would move the `php-cs-fixer` check to this build :) WDYT ?

File: .travis.yml (hunk 1 above)
Comment (chalasr) [reply .travis.yml:11]: Yeah so I moved it up for readability

**Review by ogizanagi — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (chalasr) [reply .travis.yml:13]: Oops, merged :)

Comment (ogizanagi) [reply .travis.yml:13]: 😄

**Pull request merged**